README.md

```diff
@@ -26,6 +26,12 @@ instances running inside your VMs.
 For both the modes to work, a user has to install and start Open vSwitch in
 each VM/host that he plans to run his containers.
 
+Installing Open vSwitch is out of scope of this documentation.
+You can read [INSTALL.Debian.md] or [INSTALL.RHEL.md] or [INSTALL.Fedora.md]
+for that and then install OVN related packages along with OVS packages.
```

Comment:
Yeah. That is probably the right thing to do. I did not want people unfamiliar with OVS to read INSTALL.md in entirety as that can be an exhausting process and so easy to mess up. But since the re-direction to other files are mentioned right at the top, it should be okay.